statuscake/resource_statuscaketest_test.go

```diff
@@ -240,7 +219,7 @@ resource "statuscake_test" "google" {
 	test_type = "HTTP"
 	check_rate = 300
 	timeout = 10
-	contact_id = 43402
+	contact_id = %d
```

Comment:
Don't understand what is this.

Reply:
It gets replaced by the Sprintf call in the test. None of the contact ids on my account were 43402 so I needed to swap in a valid ID.

Reply:
It gets replaced by the Sprintf call in the test. None of the contact ids on my account were 43402 so I needed to swap in a valid ID.